src/markerclusterer.ts

```diff
@@ -222,6 +236,10 @@ export class MarkerClusterer extends OverlayViewSafe {
       if (cluster.markers.length === 1) {
         cluster.marker = cluster.markers[0];
       } else {
+        // we disable the markers here only when we're re-processing the cluster marker
+        // this stops the markers from flickering on each map event.
+        cluster.markers.forEach((marker) => marker.setMap(null));
```

Comment:
OOps this one would needs to be update to to use

https://github.com/googlemaps/js-markerclusterer/blob/9476b3a62399fd124c92ed74faae6943cf573ea4/src/marker-utils.ts#LL33C1-L39C4